content/en/docs/v3.5/downgrades/downgrade_3_5.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Downgrade etcd from 3.5 to 3.4
+weight: 6650
+description: Processes, checklists, and notes on downgrading etcd from 3.5 to 3.4
+---
+
+In the general case, downgrading from etcd 3.5 to 3.4 can be a zero-downtime, rolling downgrade:
+ - one by one, stop the etcd v3.5 processes and replace them with etcd v3.4 processes
+ - after starting any v3.4 processes, new features in v3.5 are not longer available to the cluster
+
+Before [starting an downgrade](#downgrade-procedure), read through the rest of this guide to prepare.
+
+### Downgrade checklists
+
+**NOTE:** If your cluster enables auth, rolling downgrade from 3.5 isn't supported because 3.5 [changes a format of WAL entries related to auth](https://github.com/etcd-io/etcd/pull/11943).
+
+Highlighted breaking changes from 3.5 to 3.4:
+
+#### Difference in flags
+
+If you are using any of the following flags in your v3.5 configurations, make sure to remove, rename, or change the default value when downgrading to v3.4.
+
+```diff
+# flags not available in 3.4
```

Comment:
Interesting list, which versions exact the diff comes from. I think we backported some flags to v3.4, so the exact diff differs from version to version. Possibly the best solution would be to clarify how user can get the diff for their version or to specify that this diff comes from latest patch version.

Reply:
done